packages/camera/camera_avfoundation/CHANGELOG.md

```diff
@@ -1,195 +1,196 @@
 ## NEXT
 
-* Updates minimum supported SDK version to Flutter 3.22/Dart 3.4.
+- Updates minimum supported SDK version to Flutter 3.22/Dart 3.4.
+- Removes OCMock from tests.
```

Comment:
is minimum SDK bump related to OCMock removal?

Reply:
No, it's not, it's been done before in a different PR.